### PR TITLE
WE DID IT

### PR DESCRIPTION
**Pull request by ramumnesys**

### CODE REVIEW

**Comment by ramumnesys** [comment]
We did it lol

File: FoosLiveAndroid/FoosLiveAndroid/Fragments/SettingsFragment.cs [hunk 1]
```diff
@@ -127,10 +135,12 @@ public override View OnCreateView(LayoutInflater inflater, ViewGroup container,
         private void UpdateSelection()
         {
             //Extract default values from resources
-            var team1GoalSoundDefault = GetString(Resource.String.saved_team1_goal_default);
-            var team2GoalSoundDefault = GetString(Resource.String.saved_team2_goal_default);
-            var team1WinSoundDefault = GetString(Resource.String.saved_team1_win_default);
-            var team2WinSoundDefault = GetString(Resource.String.saved_team2_win_default);
+
+            //Todo: use string resources instead of enum
```
Comment (pauliuskalv) [inline FoosLiveAndroid/FoosLiveAndroid/Fragments/SettingsFragment.cs]: Remove the commented lines

File: FoosLiveAndroid/FoosLiveAndroid/Fragments/SettingsFragment.cs [hunk 2]
```diff
@@ -188,31 +198,31 @@ private void OpenSoundPicker(string soundItem, ArrayAdapter<string> adapter, Tex
...
-                            prefsEditor.PutString(soundItem, SoundAsset.GoalMario.ToString()).Apply();
-                            soundTitle.Text = goalSoundName;
+                            prefsEditor.PutString(sharedPrefKey, SoundAsset.GoalMario.ToString()).Apply();
+                            soundTitle.Text = SoundAsset.GoalMario.ToString();
                             break;
                         }
                     case (int)SoundAsset.WinMario:
                         {
-                            var winSoundName = GetString(Resource.String.mario_win_sound);
+                            //var winSoundName = GetString(Resource.String.mario_win_sound);
```
Comment (pauliuskalv) [inline FoosLiveAndroid/FoosLiveAndroid/Fragments/SettingsFragment.cs]: Remove the commented line

File: FoosLiveAndroid/FoosLiveAndroid/GameActivity.cs [hunk 3]
```diff
@@ -34,10 +35,15 @@ public class GameActivity : AppCompatActivity, View.IOnTouchListener
         private static readonly int PreviewWidth = PropertiesManager.GetIntProperty("preview_width");
         private static readonly int PreviewHeight = PropertiesManager.GetIntProperty("preview_height");
 
+        private static readonly int ColorZoneCount = PropertiesManager.GetIntProperty("color_zone_count");
+
         // A constant for upscaling the positions
         private float _upscaleMultiplierX;
         private float _upscaleMultiplierY;
 
+        private int FakeTeam1Added;
```
Comment (pauliuskalv) [inline FoosLiveAndroid/FoosLiveAndroid/GameActivity.cs]: These integers are not used anywhere

File: FoosLiveAndroid/FoosLiveAndroid/Util/Game.cs [hunk 4]
```diff
@@ -68,10 +71,11 @@ public Game(float mulX, float mulY, Activity activity, TextView ballSpeed, TextV
                 var team1GoalKey = activity.Resources.GetString(Resource.String.saved_team1_goal);
                 var team2GoalKey = activity.Resources.GetString(Resource.String.saved_team2_goal);
 
-                var team1WinDefaultValue = activity.Resources.GetString(Resource.String.saved_team1_win_default);
-                var team2WinDefaultValue = activity.Resources.GetString(Resource.String.saved_team2_win_default);
-                var team1GoalDefaultValue = activity.Resources.GetString(Resource.String.saved_team1_goal_default);
-                var team2GoalDefaultValue = activity.Resources.GetString(Resource.String.saved_team2_goal_default);
+                // Todo: use string resources instead of enum
```
Comment (pauliuskalv) [inline FoosLiveAndroid/FoosLiveAndroid/Util/Game.cs]: Remove commented lines

File: FoosLiveAndroid/FoosLiveAndroid/Util/Sounds/FilePathResolver.cs [hunk 5]
```diff
@@ -19,29 +19,26 @@ static class FilePathResolver
 
         public static AssetFileDescriptor GetFile(Context context, String argument)
         {
-            System.Console.WriteLine(argument);
-
+            //System.Console.WriteLine(argument);
```
Comment (pauliuskalv) [inline FoosLiveAndroid/FoosLiveAndroid/Util/Sounds/FilePathResolver.cs]: Remove the commented line

**Pull request merged**